README.md

```diff
@@ -146,6 +146,7 @@ If you are not a programmer, but would like to contribute, check out the [Awesom
 - [Zulip](https://github.com/zulip/zulip/labels/good%20first%20issue) _(label: good first issue)_ <br> Powerful open source group chat.
 - [cookiecutter](https://github.com/audreyr/cookiecutter/labels/low-hanging-fruit) _(label: low-hanging-fruit)_ <br> A command-line utility that creates projects from cookiecutters (project templates). E.g. Python package projects, jQuery plugin projects.
 - [django cookiecutter](https://github.com/pydanny/cookiecutter-django/labels/hacktoberfest) _(label: django hacktober special)_ <br> An implementation of Python for backend web development.
+- [Flask](https://github.com/pallets/flask/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) _(label: good first issue)_ <br> The Python micro framework for building web applications.
```

Comment:
check format for links on surronding examples

Reply:
Hi @vvscode ,

I don't understand. I already followed the rule of format in `CONTRIBUTING.md`.

Reply:
Not exactly, the link should look like this:

```suggestion
- [Flask](https://github.com/pallets/flask/labels/good%20first%20issue) _(label: good first issue)_ <br> The Python micro framework for building web applications.
```